server/queue/queue-api/src/test/java/org/apache/james/queue/api/PriorityMailQueueContract.java

```diff
@@ -199,4 +201,8 @@ default void priorityCanBeOmitted() throws Exception {
         MailQueue.MailQueueItem mailQueueItem = getMailQueue().deQueue();
         assertThat(mailQueueItem.getMail().getName()).isEqualTo("name1");
     }
+
+    default Attribute mailPriority(Object priority) {
```

Comment:
Isn't it always an int?

Wouldn't it make sense as a static method in `MailPrioritySupport` ? (this API would then be responsible of the attribute formatting which IMO makes sense)

Reply:
No, you have an error case which put a String in it